docs/ccv1_image_security_design.md

```diff
@@ -231,27 +208,29 @@ The steps are as follows:
    - If it is unconditional acceptance, skip the second and third steps; 
    - If it is a signature verification requirement, proceed to the following steps
 
-2. (if needed in policy) Download the signature claim file of the container image.
-   - If the registry support `X-Registry-Supports-Signatures` API extension, 
-     download from the registry together with image.
-   - Else, download from `sigstore` according to the config file.
+2. (if signsture verification is needed in policy) Get the signature of the container image.
 
-3. (if needed in policy) Verify the signature of the container image according to the configuration of the policy.
+3. (if signsture verification is needed in policy) Verify the signature of the container image according to the configuration of the policy.
 
 4. Pull the container image layer by layer. If an encrypted container image layer is encrypted, decrypt it.
 
 After the image is pulled, the subsequent deployment process is consistent with that of the normal container image.
 
 ## Policy
 
-Policy file (`policy.json`) is a necessary configuration file for image pulling in CCv1,
-the spec of it is defined in 
-[container/image policy.json](https://github.com/containers/image/blob/main/docs/containers-policy.json.5.md).
+The policy file here comes from the signature verification [policy configuration file](https://github.com/containers/image/blob/main/docs/containers-policy.json.5.md)
+introduced in the implementation of red hat simple signing in the [containers/image](https://github.com/containers/image) project. 
+This policy file can not only be used for red hat simple signing In the secret container design, 
+we will continue to customize and expand the content of the policy file as 
+a general signature verification configuration file compatible with different signature schemes.
```

Comment:
Are we assuming that all signature schemes are going to have a policy file of some sort? Is there any reason to allow schemes to request additional files?

Reply:
Maybe it is a better idea to use a unified policy file format in image-rs. I think there are the following reasons:

1. The policy file is distributed by the owner from the attestation-agent, which is a good channel for image-rs to accept the owner's control and configuration. The use of a unified policy file format enables the owner to determine which image use which scheme for signature verification through the policy file. In addition, this is conducive to playing more roles of the policy file on the image-rs side in the future.

2. The policy file format we currently use provides a strong configuration mechanism and has flexible scalability. We can support different signature schemes by customizing the current policy file.

3. Using the unified policy file format can make the architecture of image-rs verification signature function not too loose. Maybe this will enable us to do more optimization in the future.

Reply:
Ok, I think it's fine to start with this assumption. We might need to revisit if we encounter a signature scheme with totally different parameters.